studio/components/interfaces/Auth/Policies/PolicyTableRow/index.tsx

```diff
@@ -9,50 +9,57 @@ import Panel from 'components/ui/Panel'
 
 interface Props {
   table: PostgresTable
+  isLocked: boolean
   onSelectToggleRLS: (table: PostgresTable) => void
   onSelectCreatePolicy: (table: PostgresTable) => void
   onSelectEditPolicy: (policy: PostgresPolicy) => void
   onSelectDeletePolicy: (policy: PostgresPolicy) => void
 }
 
-const PolicyTableRow: FC<Props> = observer(
-  ({
-    table,
-    onSelectToggleRLS = () => {},
-    onSelectCreatePolicy = () => {},
-    onSelectEditPolicy = () => {},
-    onSelectDeletePolicy = () => {},
-  }) => {
-    const { meta } = useStore()
-    const policies = meta.policies.list((x: PostgresPolicy) => x.table === table.name)
-    return (
-      <Panel
-        title={
-          <PolicyTableRowHeader
-            table={table}
-            onSelectToggleRLS={onSelectToggleRLS}
-            onSelectCreatePolicy={onSelectCreatePolicy}
-          />
-        }
-      >
-        {policies.length === 0 && (
-          <div className="p-4 px-6">
-            <p className="text-scale-900 text-sm">No policies created yet</p>
-          </div>
-        )}
+const PolicyTableRow: FC<Props> = ({
+  table,
+  isLocked,
+  onSelectToggleRLS = () => {},
+  onSelectCreatePolicy = () => {},
+  onSelectEditPolicy = () => {},
+  onSelectDeletePolicy = () => {},
+}) => {
+  const { meta } = useStore()
+  const policies = meta.policies.list((x: PostgresPolicy) => x.table === table.name)
 
-        {/* @ts-ignore */}
-        {policies.map((policy) => (
-          <PolicyRow
-            key={policy.id}
-            policy={policy}
-            onSelectEditPolicy={onSelectEditPolicy}
-            onSelectDeletePolicy={onSelectDeletePolicy}
-          />
-        ))}
-      </Panel>
-    )
-  }
-)
+  return (
+    <Panel
+      title={
+        <PolicyTableRowHeader
+          table={table}
+          isLocked={isLocked}
+          onSelectToggleRLS={onSelectToggleRLS}
+          onSelectCreatePolicy={onSelectCreatePolicy}
+        />
+      }
+    >
+      {policies.length === 0 && (
+        <div className="p-4 px-6 space-y-1">
+          <p className="text-scale-1100 text-sm">No policies created yet</p>
+          {table.rls_enabled && (
+            <p className="text-scale-1000 text-sm">
```

Comment:
> As RLS has been enabled, all normal access to this table must be allowed by a policy.

I think this could be a touch simpler. But how!

— RLS has been enabled. You must create a new policy to allow access to it. 
— RLS has been enabled. Create a new policy to allow access to it. )
— RLS has been enabled. Access is now controlled by policies. Create a new one to get started.  

Wording — so hard!

Reply:
i'll go with the second one!